convert_mistral_to_gguf.py

```diff
@@ -0,0 +1,1118 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from __future__ import annotations
+
+import logging
+import argparse
+import json
+import os
+import sys
+from enum import IntEnum
+from pathlib import Path
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    ContextManager,
+    Iterable,
+    Iterator,
+    Sequence,
+    Type,
+    cast,
+)
+
+import numpy as np
+import torch
+
+from gguf.constants import MODEL_ARCH, MODEL_ARCH_NAMES
+from gguf.vocab import MistralTokenizerType, MistralVocab
+from mistral_common.tokens.tokenizers.multimodal import DATASET_MEAN, DATASET_STD
+
+if TYPE_CHECKING:
+    from torch import Tensor
+
+if "NO_LOCAL_GGUF" not in os.environ:
+    sys.path.insert(1, str(Path(__file__).parent / "gguf-py"))
+import gguf
+
```

Comment:
```suggestion
if "NO_LOCAL_GGUF" not in os.environ:
    sys.path.insert(1, str(Path(__file__).parent / "gguf-py"))
import gguf

from gguf.constants import MODEL_ARCH, MODEL_ARCH_NAMES
from gguf.vocab import MistralTokenizerType, MistralVocab
from mistral_common.tokens.tokenizers.multimodal import DATASET_MEAN, DATASET_STD

if TYPE_CHECKING:
    from torch import Tensor

```
This should hopefully also allow the CI to complete.